scripts/setup-replication.sh

```diff
@@ -43,7 +43,12 @@ if [[ "$WAL_LEVEL" == 'logical' ]]; then
 	echo "We have setup logical replication"
 elif [[ "$WAL_LEVEL" == 'replica' ]]; then
   # No content yet - but this is a slave database
-  until su - postgres -c "pg_isready"
+  if [ -z "${REPLICATE_FROM}" ]; then
+    echo "You have not set REPLICATE_FROM variable."
+    echo "Specify the master address/hostname in REPLICATE_FROM and REPLICATE_PORT variable."
+  fi
+
+  until su - postgres -c "pg_isready -h ${REPLICATE_FROM} -p ${REPLICATE_PORT}"
```

Comment:
We just need to specify the target to check which is coming from REPLICATE_FROM and REPLICATE_PORT.
We want to check if the publisher is up yet.

Reply:
LGTM